bot/skills/core.py

```diff
@@ -14,13 +14,13 @@ def add_core(upd: Updater, core_handlers_group: int):
 
 
 @run_async
-def start(update: Update, context: CallbackContext):
+def start(update: Update):
```

Comment:
Usually, if functions should expect extra args, but you do not need them, it should look like

 ```python
def f(a: int, b: str, *args, **kwargs):
    pass
```